.github/workflows/integration.yml

```diff
@@ -58,4 +58,10 @@ jobs:
           cd ${GITHUB_WORKSPACE}
           FOLDER=/builds/test_gha_dxcompiler_$(date +%s)_$RANDOM
           dx mkdir $FOLDER
-          python3 scripts/run_tests.py --test M --folder $FOLDER --project $PROJECT
+          TEST=cwl_cromwell
+          python3 scripts/run_tests.py --test $TEST --folder $FOLDER --project $PROJECT --delay-compile-errors --delay-run-errors
+      - name: Upload artifact
```

Comment:
This "Upload artifact" step should be removed when you update the branch with develop. It has been removed in develop.

Reply:
Fixed.